src/test/java/com/google/api/generator/gapic/composer/BUILD.bazel

```diff
@@ -37,8 +37,9 @@ java_proto_library(
 
 [java_test(
     name = test_name,
-    srcs = ["{0}.java".format(test_name)],
+    srcs = glob(["*.java"]),
     data = [
```

Comment:
This change is for including `ComposerConstants.java` as src, not only the tests files.

Reply:
How about:
```
srcs = [
    "{0}.java".format(test_name)", 
    "ComposerConstants.java",
]
```

Reply:
Thanks! That should also work, while if we add more helpers, those files have to be explicitly included. By using `glob(["*.java"])` we can avoid modifying this even if more files are added. WDYT?

Reply:
Done! Talked offline and for suggested option, we can avoid including other test files.